app/services/healthcheck.rb

```diff
@@ -0,0 +1,60 @@
+require 'sidekiq/api'
+
+class Healthcheck
```

Comment:
These should probably all `rescue`, just in case something goes wrong, so we can give some sort of default healthcheck output even if everything downstream is broken? (We just had exactly this on preprod.)